### PR TITLE
docs: Downstream tools

### PR DESCRIPTION
**Pull request by igorlukanin**
Preview: https://deploy-preview-4197--cubejs-docs-staging.netlify.app/config/downstream

- [x] Update logo in README
- [x] Add schema to README
- [x] Add schema to docs/Introduction
- [x] Add docs/Connecting to Visualization Tools
- [x] Add docs/individual pages for viz tools
- [x] Add docs/viz tools' logos

### CODE REVIEW

File: docs/content/Configuration/Connecting-to-Downstream-Tools.mdx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+</Grid>
+
+<InfoBox>
+  If you'd like to connect to a tool which is not yet supported,
+  please <a href="https://github.com/cube-js/cube.js/issues">file an issue</a> on GitHub.
+</InfoBox>
+
+## Frontend integrations
+
+Cube provides  integration libraries for popular frontend frameworks:
```
Comment (rpaik) [inline docs/content/Configuration/Connecting-to-Downstream-Tools.mdx:42]:
```suggestion
Cube provides integration libraries for popular front-end frameworks:
```

File: docs/content/Configuration/Connecting-to-Downstream-Tools.mdx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  <GridItem
+    url="frontend-introduction/vue"
+    imageUrl=""
+    title="Vue"
+  />
+</Grid>
+
+## APIs references
+
+All integrations above are powered by following APIs. If you're a data engineer, please explore SQL API. If you're an application developer, check out REST and GraphQL APIs.
```
Comment (rpaik) [inline docs/content/Configuration/Connecting-to-Downstream-Tools.mdx:64]:
```suggestion
All integrations above are powered by the following APIs. If you're a data engineer, please explore the SQL API. If you're an application developer, check out REST and GraphQL APIs.
```

File: docs/src/pages/index.tsx [hunk 3]
```diff
@@ -30,65 +30,63 @@ class IndexPage extends Component<Props> {
...
-              practical examples to learn how Cube.js works.
+              practical examples to learn how Cube works.
             </p>
           </Col>
         </Row>
         <div className={styles.mainTabs}>
           <Row>
             <MainTab
-              title="Getting Started"
+              title="Get Started"
```
Comment (rpaik) [inline docs/src/pages/index.tsx:50]: I think "Getting started" sounds friendlier. "Get started" almost sounds like a command.

**Comment by rpaik** [comment]
A few suggestions 😄

**Comment by igorlukanin** [comment]
Thanks, @rpaik! Very helpful 🙌

File: docs/src/pages/index.tsx (hunk 3 above)
Comment (keydunov) [inline docs/src/pages/index.tsx:50]:
```suggestion
              title="Getting Started"
```

**Pull request merged**